onedal/cluster/kmeans.py

```diff
@@ -87,9 +87,11 @@ def _tolerance(self, rtol, X_table, policy, dtype=np.float32):
         if rtol == 0.0:
             return rtol
         # TODO: Support CSR in Basic Statistics
```

Comment:
Should we remove line 89 in this PR?

Reply:
I think not now. Because currently CSR inputs are now supported by K-means and `is_sparse == False` is passes to Basic Statistics algorithm.

I'll remove this when CSR support is added to K-means.

Reply:
As long as this ends up in a ticket or somewhere it isn't forgotten (outside the TODO), I'm cool with it.

Reply:
This will be addressed in KMeans OOP PR, no need for a separate ticket.